django/views/generic/dates.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 
 from django.conf import settings
+from django.contrib.admin.utils import get_fields_from_path
```

Comment:
Yes, this feels like a meta thing. I'm a bit surprised if something similar doesn't already exist but perhaps there are other helps that could also be consolidated.